onnx/optimizer/optimize.cc

```diff
@@ -11,6 +11,10 @@ GlobalPassRegistry Optimizer::passes;
 Optimizer::Optimizer(
     const std::vector<std::string>& names,
     const bool fixed_point) {
+  std::cout
+      << "WARNING: ONNX Optimizer has been moved to https://github.com/onnx/optimizer. "
```

Comment:
Also add a statement stating that all further enhancements and fixes to optimizers will be done in this new repo.